packages/api/src/routes/medical/network.ts

```diff
@@ -0,0 +1,44 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import status from "http-status";
+import {
+  getHieDirectoryEntriesByFilter,
+  getHieDirectoryEntriesByFilterCount,
+} from "../../external/carequality/command/cq-directory/get-cq-directory-entries";
+import { requestLogger } from "../helpers/request-logger";
+import { asyncHandler } from "../util";
+import { dtoFromHieDirectoryEntry, NetworkDTO } from "./dtos/networkDTO";
+import { networkGetSchema } from "./schemas/network";
+
+import { PaginatedResponse } from "@metriport/shared";
```

Comment:
All outside imports "event from metriport" usually go towards the top and local file imports are below them

Reply:
addressed